docs/signalfx-smart-agent-migration.md

```diff
@@ -0,0 +1,118 @@
+# Migrating from the Smart Agent to the Splunk distribution of OpenTelemetry Collector
```

Comment:
"Migrating from the SignalFx Smart Agent to the Splunk Distribution of OpenTelemetry Collector"